app/models/product_safety_alert.rb

```diff
@@ -0,0 +1,33 @@
+class ProductSafetyAlert < Document
+  validates :product_alert_type, presence: true
+  validates :product_risk_level, presence: true
+  validates :product_category, presence: true
+  validates :product_measure_type, presence: true
+  validates :product_recall_alert_date, allow_blank: true, date: true
+
+  FORMAT_SPECIFIC_FIELDS = %i[
+    product_alert_type
+    product_risk_level
+    product_category
+    product_measure_type
+    product_recall_alert_date
+  ].freeze
+
+  attr_accessor(*FORMAT_SPECIFIC_FIELDS)
+
+  def initialize(params = {})
+    super(params, FORMAT_SPECIFIC_FIELDS)
+  end
+
+  def taxons
```

Comment:
Not sure we need to define the `taxons` method here as the parent model has the same method, with the same output (empty array): https://github.com/alphagov/specialist-publisher/blob/main/app/models/document.rb#L84-L86.